### DISCUSSION TITLE
Align navbar items to left

### DISCUSSION THREAD

**Discussion by httpanand** [twbs/bootstrap#35346] [General]
![image](https://user-images.githubusercontent.com/87514488/141116267-82ff2fc2-aeb6-4105-8aaf-56193df604f1.png)

So... is there any way to make the `home` , `about` , `contact` left align . Is there any default classes in bootstrap ?  I tried with text-align property but was unsuccessful ! I would be grateful for your prompt reply .

**Comment by ascensus-mdb** [comment]
Like this? 
https://mdbootstrap.com/docs/standard/navigation/navbar/#subsection-left-aligned

**Reply by httpanand** [reply]
Yess .Will i be able to make it with bootstrap alone ?

**Reply by ascensus-mdb** [reply]
Should be fine ;)
